README.md

```diff
@@ -25,7 +26,7 @@ To install all the available dependecies run:
 
 To run this application locally :
```

Comment:
- [ ] Something to improve on the readme file, is to talk about the API and add instructions about how to use it. Please, include all that's needed for running the app with the API(s) it consumes. It should have instructions about where and how to generate any necessary keys, credentials, how to set, store, and generate them (if necessary).

Reply:
Thank you for letting me improve my README file.